proposals/working-group/Comparisons.adoc

```diff
@@ -0,0 +1,36 @@
+= Pros/Cons of Each Proposal
+
+== MPWG
+Pros
+
+* Flatter governing structure as steering committee, marketing committee and specification committees share the same membership.
+* Addresses IP gap in existing MP development
+* Provides complete autonomy of how MP evolves as governing structure is not part of a non-MP specific working group
+* Provides a lower barrier to entry in terms of both cost and process for the MP technology
+* Continues a model that has produced multiple independent implementations of novel technologies over a number of years
+* Allows for large companies to sponsor specification development at a zero cost entry point
+* Less restrictive use of compatibility and logo usage
```

Comment:
Is that defined anywhere. How do you maintain quality over the trademark?

Reply:
The thinking here is  that MP  logo could be used on individual spec implementations, not just at the  platform level. Java EE lost a lot of leverage by not letting the compatibility logo be used with just servlet, for example. How many developers know that a tremendous amount of Java EE technology underlies Spring?

This is part of the benefit of having a separate MPWG. It can make decisions that are substantively different than Jakarta EE. Some  decisions may work out better than Jakarta EE, some may come out worse. However, in the end both MicroProfile and Jakarta EE can try different approaches and see what works. Then the projects make adjustments along the way. Pragmatically speaking, I don't think that will happen under a single steering committee.

Reply:
That's not in the proposal under review and is not inherent to the MPWG proposal.

Reply:
It is an opaque detail in the "Marketing Encourage brand proliferation" item.